tests/pytests/functional/modules/test_mac_timezone.py

```diff
@@ -0,0 +1,225 @@
+"""
+Integration tests for mac_timezone
+
+If using parallels, make sure Time sync is turned off. Otherwise, parallels will
+keep changing your date/time settings while the tests are running. To turn off
+Time sync do the following:
+    - Go to actions -> configure
+    - Select options at the top and 'More Options' on the left
+    - Set time to 'Do not sync'
+"""
+
+import datetime
+
+import pytest
+
+pytestmark = [
+    pytest.mark.flaky(max_runs=4),
+    pytest.mark.skip_if_binaries_missing("systemsetup"),
+    pytest.mark.slow_test,
+    pytest.mark.destructive_test,
+    pytest.mark.skip_if_not_root,
+    pytest.mark.skip_unless_on_darwin,
+]
+
+
+@pytest.fixture(scope="module")
+def timezone(modules):
+    return modules.timezone
+
+
+@pytest.fixture(scope="function", autouse=True)
+def _setup_teardown_vars(timezone):
+    USE_NETWORK_TIME = timezone.get_using_network_time()
+    TIME_SERVER = timezone.get_time_server()
+    TIME_ZONE = timezone.get_zone()
+    CURRENT_DATE = timezone.get_date()
+    CURRENT_TIME = timezone.get_time()
+
+    timezone.set_using_network_time(False)
+    timezone.set_zone("America/Denver")
+
+    try:
+        yield
+    finally:
+        timezone.set_time_server(TIME_SERVER)
+        timezone.set_using_network_time(USE_NETWORK_TIME)
+        timezone.set_zone(TIME_ZONE)
+        if not USE_NETWORK_TIME:
+            timezone.set_date(CURRENT_DATE)
+            timezone.set_time(CURRENT_TIME)
+
+
+@pytest.mark.skip(
+    reason="Skip until we can figure out why modifying the system clock causes ZMQ errors",
+)
```

Comment:
```suggestion
@pytest.mark.skip(
    reason="Skip until we can figure out why modifying the system clock causes ZMQ errors",
)
```
These errors still occur?